dd-sdk-android-core/src/main/kotlin/com/datadog/android/core/internal/CoreFeature.kt

```diff
@@ -613,6 +620,19 @@ internal class CoreFeature(
             executorContext = "storage",
             backPressureStrategy = backpressureStrategy
         )
+        // TODO RUM-9851 Switch to the executor which is aware of backpressure, but only logs it
+        @Suppress("UnsafeThirdPartyFunctionCall") // all parameters are safe
+        contextExecutorService = ThreadPoolExecutor(
+            // core pool size
+            1,
+            // max pool size,
+            1,
+            // keep-alive time
+            0L,
```

Comment:
why not keeping it alive for a while ?

Reply:
This is to align with defaults of the `Executors.newSingleThreadExecutor` call ([link](https://cs.android.com/android/platform/superproject/main/+/main:libcore/ojluni/src/main/java/java/util/concurrent/Executors.java;l=197-202))